packages/actions/src/helpers/s3.ts

```diff
@@ -0,0 +1,320 @@
+import { Functions, httpsCallable, HttpsCallable } from "firebase/functions"
+import mime from "mime-types"
+import fs from 'fs'
+import { ETagWithPartNumber, ChunkWithUrl } from "../../types"
+import fetch from "@adobe/node-fetch-retry"
+import https from "https"
+import dotenv from "dotenv"
+
+dotenv.config()
+
+/**
+ * Return the bucket name based on ceremony prefix.
+ * @param ceremonyPrefix <string> - the ceremony prefix.
+ * @returns <string>
+ */
+export const getBucketName = (ceremonyPrefix: string): string => {
+    if (!process.env.CONFIG_CEREMONY_BUCKET_POSTFIX) return ''
```

Comment:
Give a short explanation for this error.

Bonus: use a code (e.g., throw new Error('S3-001: short explanation here')).